src/components/ShowCard.tsx

```diff
@@ -68,6 +68,7 @@ export default function ShowCard(props: MovieCardProps): JSX.Element {
 							</div>
 						)}
 					</div>
+					<Link to={`/details/${props.details.id}`} state={props} >Details</Link>
```

Comment:
I think the entire show card should be clickable. rather than a dedicated button.